l2_naming/naming_1.md

```diff
@@ -2,42 +2,40 @@
 
 Write the full name or what you think it is
 
-----
- 
-* msg = "message"
-* exc =
-* srv =
-* num =
-* ci =
-* cd =
-* ex =
-* arg =
-* param =
-* dict =
-* api =
-* app =
-* cli =
-* orm =
-* dto =
-* dao =
-* repo =
-* ver =
-* db =
-* sla =
-* rfc =
-* imho =
-* tbd =
-* n_a =
-* wtf =
-* sso =
-* cto =
-* etl =
-* sql =
-* bi =
-* ux =
-* gui =
-* dev =
-* sdk =
-* lib =
-* uid =
-* cat =
+msg = "message"
+exc = "execute"
+srv = "server"
+num = "number"
+ci = "command interface"
+cd = "current directory"
```

Comment:
en UNIX, cd é "change directory", pero neste caso ia por Continous Deployment (CD en maiúsculas sería máis apropiado)